webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/campaigns/CampaignEventRepository.scala

```diff
@@ -83,7 +85,7 @@ class CampaignEventRepositoryImpl(doobie: Doobie, campaignSerializer: CampaignSe
             CampaignEventId(d._1)
           , CampaignId(d._2)
           , d._3
-          , CampaignEventState.parse(d._4).getOrElse(Scheduled)
+          , d._4.fromJson[CampaignEventState].getOrElse(CampaignEventState.parse(d._4).getOrElse(Scheduled))
```

Comment:
I think the default in case of problem when unserializing should be either an error or `Skipped("error when reading from base even state: ...")`

Reply:
In fact i should even remove the first "getOrElse" ! nice idea the Skpped for error case

Reply:
I kept it first getOrElse to migrate my dev env but in fact i should not keep it